eng/common/scripts/GeneralReleaseNotesLogic.ps1

```diff
@@ -0,0 +1,129 @@
+. (Join-Path $PSScriptRoot common.ps1)
```

Comment:
Lets move these functions to the azure-sdk repo as they don't need to be in common.